RELEASE_NOTES.md

```diff
@@ -1,5 +1,43 @@
 # Release Notes
 
+## Changes in Splinter 0.5.14
+
+### Highlights
+
+* Splinterd will not start if there are database migrations that need to be run
+  and will give information about how to upgrade the database.
+
+* User configurable logging with stdout, stderr, file, and rotating file log
+  targets as well as fine grain control over where each log statement goes.
+
+### libsplinter
+
+* Replace the use of a Vec<u8> to represent a public key in the
+  PeerAuthorizationToken, ConnectionAuthorizationType, Identity, and in the
+  AdminService.
+
+* Add any_pending_migrations to postgres and sqlite migration modules
```

Comment:
Add code back-ticks to the `any_pending_migrations`

Reply:
fixed